hardhat.config.js

```diff
@@ -75,6 +76,13 @@ module.exports = {
       viaIR: withOptimizations && argv.ir,
     },
   },
+  warnings: {
+    ignore: {
+      '**/*': {
```

Comment:
isn't `*` enough?

Reply:
`*` matches `A.sol` but it doesn't match `c/d/A.sol`.

Reply:
I updated the plugin to allow `*`.